ConsoleApp/Program.cs

```diff
@@ -0,0 +1,95 @@
+﻿static bool StringCompare(string str1, string str2) => str1 == str2;
```

Comment:
we've discussed during the lesson that it's denied to use equality operator and build-in methods

Reply:
Sorry for misunderstanding. I`ve already fixed it. Thanks!